etc/kayobe/cephadm.yml

```diff
@@ -12,7 +12,10 @@ cephadm_ceph_release: "pacific"
 cephadm_image: "{{ stackhpc_docker_registry if stackhpc_sync_ceph_images | bool else 'quay.io' }}/ceph/ceph:{{ cephadm_image_tag }}"
 
 # Ceph container image tag.
-cephadm_image_tag: "v16.2.11"
+cephadm_image_tag: "{{ 'v17.2.6' if ansible_facts['distribution_release'] == 'jammy' else 'v16.2.11' }}"
+
+# Ceph custom repo workaround for Ubuntu Jammy as there are no official ceph repos for jammy.
+cephadm_custom_repos: "{{ true if ansible_facts['distribution_release'] == 'jammy' }}"
```

Comment:
```suggestion
cephadm_custom_repos: "{{ ansible_facts['distribution_release'] == 'jammy' }}"
```

Reply:
Previous comments on this: https://github.com/stackhpc/stackhpc-kayobe-config/pull/512#discussion_r1192204572

> Dont want to change it to true if its been set false somewhere else. Only want to set to false or leave it as it is.